osc/build.py

```diff
@@ -145,7 +145,10 @@ def __init__(self, filename, apiurl, buildtype = 'spec', localpkgs = [], binaryt
         else:
             self.release = None
         if config['api_host_options'][apiurl]['downloadurl']:
-            self.enable_cpio = False
+            # Formerly, this was set to False, but we have to set it to True, because a large
+            # number of repos in OBS are misconfigured and don't actually have repos setup - they
+            # are API only.
```

Comment:
Any idea if it is possible to detect such misconfigurations?

Reply:
Not really. But major repos like the leap 15.3/4 and suse/sle bits are misconfigured for most builds.